plugin/cr/src/test/java/org/opencds/cqf/ruler/cr/r4/provider/MeasureEvaluateAdditionalDataIT.java

```diff
@@ -0,0 +1,70 @@
+package org.opencds.cqf.ruler.cr.r4.provider;
+
+import org.hl7.fhir.r4.model.Bundle;
+import org.hl7.fhir.r4.model.IdType;
+import org.hl7.fhir.r4.model.MeasureReport;
+import org.hl7.fhir.r4.model.Parameters;
+import org.hl7.fhir.r4.model.StringType;
+import org.junit.jupiter.api.Test;
+import org.opencds.cqf.ruler.cql.CqlConfig;
+import org.opencds.cqf.ruler.cr.CrConfig;
+import org.opencds.cqf.ruler.test.RestIntegrationTest;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT, classes = {
+	MeasureEvaluateAdditionalDataIT.class, CrConfig.class, CqlConfig.class }, properties = {
+	"hapi.fhir.fhir_version=r4"
+})
```

Comment:
Still missing an annotation to reset context per-test. We need to add that and document it as the reason these additionalData tests are separated out. Unlike the rest of the tests, the success of these tests is either order dependent or requires resetting context after each test.

Reply:
Updated content (https://github.com/DBCG/cqf-ruler/wiki/Contributing) in Testing section.